test-e2e/tests/account_test.ts

```diff
@@ -16,7 +16,7 @@ const lastName = 'Tester';
 Feature('account').retry(3);
 
 Before(({ I }) => {
-  I.useConfig('test--subscription');
+  I.useConfig('subscription--config');
```

Comment:
If you're renaming all of the files, might as well make them as succinct as possible by dropping the `--config` along with the `test--`